code/modules/martial_arts/hokuto_shinken.dm

```diff
@@ -0,0 +1,115 @@
+#define SOKIN_JIZAI_KYAKU_COMBO "HHD"
+#define GOSHI_RETSU_COMBO "DDHD"
+#define GEKITSUI_SHI_COMBO "HDHDH"
+
+/datum/martial_art/hokuto_shinken
+	name = "Hokuto Shinken"
+	deflection_chance = 100
+	help_verb = /datum/martial_art/hokuto_shinken/proc/hokuto_shinken_help
+
+
+/datum/martial_art/hokuto_shinken/proc/check_streak(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(findtext(streak,SOKIN_JIZAI_KYAKU_COMBO))
+		streak = ""
+		Sokin_Jizai_Kyaku(A,D)
+		return 1
+	if(findtext(streak,GOSHI_RETSU_COMBO))
+		streak = ""
+		Goshi_Retsu(A,D)
+		return 1
+	if(findtext(streak,GEKITSUI_SHI_COMBO))
+		streak = ""
+		Gekitsui_Shi(A,D)
+		return 1
+	return 0
+
+/datum/martial_art/hokuto_shinken/proc/Sokin_Jizai_Kyaku(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	A.do_attack_animation(D)
+	if(D.stat || D.weakened)
+		return 0
+	D.visible_message("<span class='warning'>[A] delivers two swift kicks to [D]\'s neck!</span>", \
+					  	"<span class='userdanger'>[A] kicks you twice in the neck!</span>")
+	playsound(get_turf(A), "swing_hit", 50, 1, -1)
+	D.apply_damage(15, BRUTE)
+	D.Weaken(10)
+	return 1
+
+/datum/martial_art/hokuto_shinken/proc/Goshi_Retsu(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	A.do_attack_animation(D)
+
+	var/obj/item/organ/external/organ = D.get_organ(A.zone_sel.selecting)
+	if(!organ)
+		return 0
+	else if(A.zone_sel.selecting == "l_hand" || A.zone_sel.selecting == "r_hand")
+		A.visible_message("<span class='warning'>[A] strikes [D]'s [organ.name] with two extended fingers!</span>")
+		admin_attack_log(A, D, "exploded [organ.joint].", "had his [organ.joint] exploded.", "exploded [organ.joint] of")
+		playsound(get_turf(A), "strike_hit", 50, 1, -1)
+		organ.droplimb(0, DROPLIMB_BLUNT)
+		return 1
+	else
+		playsound(get_turf(A), "punch", 50, 1, -1)
+		D.apply_damage(15, BRUTE)
+		A.visible_message("<span class='warning'>[A] strikes [D] with their closed fist!</span>")
+		return 1
+	return 1
+
+/datum/martial_art/hokuto_shinken/proc/Gekitsui_Shi(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	log_and_message_admins("[A] gibbed [D] with Hokuto Shinken.")
```

Comment:
LAMA prepends user automatically. Change this to:
`log_and_message_admins("gibbed [D] with Hokuto Shinken.", A)`

Reply:
Probably should be log attack, or change `D` to `key_name(D)`.